### ISSUE TITLE
Status WA not working since update

### ISSUE THREAD

**Issue by Scabbrox** [WeakAuras/WeakAuras2#2107]
## Describe the bug

A weak aura that shows when an essence is on or off CD is no longer working since the update.

**Do you have an error log of what happened?**
It doesn't give an error, it just doesn't change the OFF CD icon to the On CD one.

**Screenshots**

**Did you try having WeakAuras as the only enabled addon and everything else (especially something like ElvUI) disabled?**

**Which version of WeakAuras are you using?**
Latest version as of April 16th 2020
**Are you on World of Warcraft Classic or Retail?**

- [ ] Retail

**Was it working in a previous version? If yes, which was the last good one?**
The previous version was fine.

### Additional Info

I use this aura which simply tracks when Lucid Dreams is available and when it is on CD.  When it's available is shows a colour icon and when its on CD it shows a black and white one.  Worked fine until yesterday but seems that the show when on cooldown and off cooldown options do nothing.  It just shows the spell as available all the time and never shows the on CD version.

https://wago.io/KKcfXqr0U

**Comment by mrbuds** [comment]
This is working fine for me using your aura

**Comment by Scabbrox** [comment]
> This is working fine for me using your aura

I'm not talking about the bar, that works fine.  What's not working for me is the icon part of it, the 'Ready' not on CD icon is always there, the grey one never displays.

**Comment by mrbuds** [comment]
Your aura doesn't have a bar, it's just an icon, and it's never grey because you didn't set any option to make it grey

**Comment by mrbuds** [comment]
Or you'r not talking about the aura you linked

**Comment by Scabbrox** [comment]
Sorry, you're right this isn't the right aura, my bad was in a chat with someone on the discord and I seem to have mixed the links up.

**Comment by Scabbrox** [comment]
I'll try and find the aura concerned and report back.

**Comment by Scabbrox** [comment]
Ok here is the correct aura: https://wago.io/-eH5DDohS

I'm using a modified version of this (no bar), with larger icons.  Essentially what is happening for me is as I described above, the coloured icon that displays of the spell when it's available and not on CD shows all the time and the second grey icon that is supposed to display when the spell goes on CD doesn't show at all.

**Comment by mrbuds** [comment]
Please export the aura you actually use

**Comment by Scabbrox** [comment]
Hoping this works

`!T336YTTXsc)UOVkLj)IenbiPUKeNQiLOSPxjkDeOSDIllrqaqsebcWaakz6nHVfBT7dW5V7P2Q2FU7JZE2A33IT7Egam4kb1fNtoXhLdnbMz6PNE6PN(2m8kPRgCLI3vkY1K2Rw9RuMDLIo(FWBN46C3vkx07LVcEyKJRUHBhvTB0DDMFLshlZp9jvx9vdh44y5BcVsDH)uh3ZM7B6y7baW)kLLNnESNH)vhVZEnLBCqJg728a5w72S5ExnENMTW(WuFWY5gq)CuqNCOJLJl0((sx1V(v9LPpBqF2eFjaxV5QAgyrklCTUszQV)CVV55p)o1jo1mDE(4B)0jt)zLxD5ZBaOLwacP45R66ZWmtBt(3gdF1Bk9D4jFxZjtmC9y9)GWxGFv3y0IXJzO7R6EY5hF5jq50JQlCvLr8YWYQNoF07TyKXTg2(kqRm)4vkxFyBLbxRmO9fdIk8CxdQqLZ7EcaoB1zg8Mtfd9KHQL)uyKIySYCl1La24JphHA(mKxZXAXmBfgXrchK(gU2QwVbgqanaiy7dOQT5mvKKisr4ddBhB4tDyWGv4A2l9m0CSHXKIHQNbqpmSNa4dmJqVyqudbmyMQP9Ji4cNBEKaiasvlZj2xPCy3(d6EbraiAiWDPCBazcAQLJkTiaMr1W)D2cGjpGjrZs1Zl1B9m)KrYxcVEeFnJiBTeXwlrS1seBD9ATObSlm)dJbvFvAgKnzzC1ia8AQwSPv2Ye8DSV1vFcmeLwn8CZpAalhCnMadd2axFjWrzQbRLxmpO(kiIIlESmNb8u9H1HUo(mEcKo45a8eCATQTgSO(ChtKvmGOb0BGUz7FmS2cgtkx0(OExQGZewJ5v9KUhhk1GleOV0UyvMbKzaBRvVgGb(oAHu99R3Oo07MG8hfn1rJEM3QHTDNbF21ZdwkyQAbtYlWYp8hFvf)pD7KLTMXKH8wtDCoVvacFmshzuaLdVOB3(esp2CcBPfdV6ztOf0NQUANeW2Sl8SRQU5cpGgHi0hdfITRuR9K3f(Vg1LpqU5(OqSMeK1nfK6bYnJwdUWZ4eImpI4CcLROTWZ3z2aJpIZ6lSjHuvQU61RgY(ZC8QHV1q9M2aZUxn(sDfywY49OWMRnSVTMP(hckX79sFy1q)Pg2rGi4pxd)fU2RgABAT9QHx4SWwVsim88DFJQ1cdHU2WYZyJ7(g5297SJPN3cJvdVZ0F6QHaNJ2nWK6yxNzWUhnb8YXF1qvR7uxcV2BQZDM2tae1ZbAc0mTPgA3qaF1qua8QH6MUgA(wlRL)yfyfHQurA3TFB7RFPH)LG0ZoqJRSftc6wBlVxRwnRxT62eHry0BRV61Whr7ErCizVoWXs35oBL7mboToItQxoxhx3QSi4FZAFQxj1EXBV5hU(p9CzGnYe4JqGu4wqCrFWcO6eV1102cDcxOFuqfgLZ(v4RuaA8zam8O)TnW7DRrGiw2qi2grP2hRB)Ux8YE)y3GAF9utDeaMZm0tUjiFxSdDMnsfMMpXzc)DWA5dp70oThC9jN9YR7(gGME9L9pU3jaPT7rbGoC8K5IelhqOiWA441HymEbW0MZ09QH8j84lXcBA28U8oiGSdqX4JZhaJtQV4mzT2UeDB185vZa6P4gH3K7Fb0Q(Wu2vkIVrGQr0COhQvR2drkt(lZdRV7cqfeGf4q26wGongKCNCmjIbHnDQQ3aNtxZKX1iDchniN92SN9Cw4Qz8Yl7De)f59)zmNhf8nKKrOiqws3HiUf2tq1G6TfPNgxtUlp8WUkkBbdayejGdSQImd4tHmdvZBGLya6PF9CxhT1Wo3sUzJ9QMpW2zN3Q6oYfMCnCxn85RgEOs(vghPCsdd5L3vwAxqLch3efiT7Es1BTUrsjyk8DxySEaSZo8wGyiBWdFh2Ny(TZbgllu7nD(eGQ(pbC9W6k1jWS1eqvuSO5ylanQm9xFVXNam9iOZNd75112y2YiA)wGsZtm83QAPrFydnaQKstA4UygAQGSB4loGMnWGrt1gFBIrvzGUQ9Z8J2IKWlo9HtRGnv8DDSSqsHoUlSVdVBSNR5Hyf0Zt8knTbxefms4llwnSNxp7E8xwP6MWwatvWe0aA6bHeN0tZNvI3pR(fO85G9klzSLzukag9QbCdb9HpZavLbx8(To)nNV1hynpNY7cLVw8pqYDX8WXKZL6Vcw4EQdW2fUYZdxSH7DQ5eXDtdrMYfWWzbuxdpFUHDfVmpVLKjiAUGwAg6uf8EpxcqkkxSkTv3pAOTW3yRpiu)YjMGZBPoY0Y0F5H048fcDeiceFzfoCRwAiYwfCUZD4sb(6z6jrjPsLaEXOCpAJ7yIFdK6VznnCjvcAx9YdIa99JrS(Eeeps4sZnezE49iSqykiCXw3zwf56WCCJdQIRSWLp4lHwm2hhMObgJvnjXJOKrGRrdvdJMr5l5kpUBxcH2LQsXeqc8z2SHvLSEpUKeuRD4xhNmuIDpgqMuzy5Chm57AAZ36ymcr04HjtWnUkbGOgxbmLeQFnO5a5VYw)YQH0kKVz1WFr74JRxFCRXJ)kVFXfBtqH9oIx8EQqfelgSuLK0rJsELgpEmunHkHvjSS61dkdH(GlwnSRYGEN2EqVZ6hGeJXQfwPloFauWx5T12HAdUDGgIPj1BlSaz7ugqVDoBRvDtLUim1(9V4ESkmfMLpRdOvOeY68)pVTKFyCZzwySxgzSn3ssMrOCJgZXhPOzVQJ8CChDQJoz3DmFJcpXBpZG0m8vAF5e2vxORAJByCGXRNFs7FO7fx39pDzVZpfnE9Wx1U)lJSCnW7KGXulGQtQOfoYszEMatsOkV06WP4IZzQ2lrpTFkxtwpMGRPQ3kiRmCgKPwaXM6jc4uMzL20SGTnV1Gw)eFFKWcNNFHX45ekMgl3aRrULPY5CvtxprUsvYlemKg4i1DY1(V5yDiZraOHU9Q4kAp2HQ6Hx3(tgUM(gDNrfyO3Z3ygQAX5b1Rch0zSILWFu3x944FyFKdolUcxNBA1ETAjFatUTWRAiL(vYBOcue2GUFQmJySEvW3Cc0yKj8BoeSw034yxNzD)5fMZNHliSC8RC77L)a6Nmo5zBA)M1ItXyf47naBmOnTsiA((T0n80CnPygT1haaV1xP)1BvsjMrDaZ(7WhjBb2y5Mcm9bF9RfG6gaN5rWzEk4meucPs0tGO361Au9EiExyvC5SL(XqwDkSiT1tceW1v9ZdRorNy7eKiUxzVjqwc1BKuOoZrKxPWCbhTXXlXaHCTMd0y0fPe)OHx2oenWfOOB8hyGAw6aCDNAozkfyhCBi(WHIfZACSkpeHyRyoM9qkksJwZMpyZere(yXph37MG4HT8Ahy7CvkqnV4ffUPAabRtmN4srRkSxJVvRn5YXKehfnNzJmTnoLrDz0JtGf9ESn1jY)H0CapejKRl7K1SQIUP3pr7sERWMWChTZQ5jotm1e2l1xy1uOL6u8rq5R(OpmxNlvZ2nRS9O49kP7r)DKQZcNCgrA9gdJ5TXai6FbkFLMQJIqwuein1UzPOF7bZaN4A45bZqAMmW1ui6KyScaI4fuu(eIsaip9AVPQ6o39oHaxqV2NcZ0xPfRwLio7SAhHZTVCWz83QUaMaauwJf6nL2WZ8Ia2kJGiYTBZGyN8djrk0zDMJxgrjOGqAWcjw)Z63nyLdmyzdaQzAPr7AhSB992VzdPg7j3Q5bcHvnAum2HgaAAM6aZpUzjqE9Wh0wgJSKcpVZXv)TUQZVs5THFJk4wyYzKfPakJKKvmIc7CEuxdJLuQjjSsSvZND5Gt6bJ)afvNawODSRXpVGqwy8k3sG2GLYKTzfmPoI1gg0gTW33X(mG)bw0YkiiyNyqKWNfhZtSIZAKow1erfR27s0SW(DkWsBdCWeJl(gLWGxtDpiBakBF2tDccNDOi81Yjd21vJmX7V3yOL4m07FqDPiM6mhfmg6JCCn9o)87jhCAwUm4K71VpyNZGZopfZSuZCzMJwB4J4DjLpj)fbupMcO2GP3oNnyWzNUjIReNHbWm1G26VFR6rjpdQtblxz6WEanmNKoX12qJcAEq204bwiaau5TTV4IENDrY0RHNjo8AHYsIvo1zuuMj9xYozCYih1eZwgke8GIvEGn7UyC77Ki7o45UbzlVGYMygdDRQRPksqiTDWNb975tWAtvTNyebayhEqNm)LX6RaP9Sg2rSwadwn5AKiwFbhyKgpKYfpQxmEG9qJAXMy9JsRmwAoq5z0OOeIbzfeZQMOC75GAab5oEo5GSqwZNYsOPp5aQVrjubUCCuUzaucvkxn8UNJ5bKpxQFakHRfpY0dhV6i0sN)u5KnqfMvGHQzUZ96)T617SZrN1)zdam(OEWND6EYzVD1Wl7FsxfLvd)HZUC1W)H(4RE7RApGEXZUO7QHhDwV(VK1(7x)U6154446ILKx85lW)qz7trrOgzyhv0)4VIyZo7Oelcwm)Zt(191zhnfwJJ0Fxi6kjlkSks72qUU0hcTeEB0hA8MLr1L3xQE965uDGp7uqCKQMBMnfeJFW(51u5SAH0Eh0QvETOrul(1Tfg0sYYh0mrJGvcoU0(c4Y9BmeR(EsTsv7xPoJ50fXk2C30GvXsDMyD2D)KeN3o1016otr7)FVCDzOMPawVzapNofV73yIzh4sH2S3EYPAWfXCRW73Vz6rYHwgKJo5SuV0vDepDWyre1zCuKsX6RbwMTA4OLRg22t9g17ufy24rsKXBhyWy6WkIUxIslTWCFG8LuO5KXqAUZ30Yi0BcfnafsrLd4aXStEHlwCn)2eomfD4i2wMtmd97yLq4LJxhr0pOD18dtELWOFIYclFAPemOdHi(LVTKE2NpNHPLaBu4GqAHRljAI7GYvdhWDRnnFtZFAolC9n8wYMFb5qwoJ88LLz1jvQ0DZjUV7WlV9DcZ1I(bp(eE6uiZZYXNl8rA7gB36xZW3vIUOosixYz6f2(8GnlLXSPO7VRqDAXo7gRsGVUZTsMOtCnTVjG7QNn6GhyTxpEbIr7MbXQzZ0ebPsLPsu1FZjIjdxZTreaEp5q9a4vuqW4WYI7fAwcLug3sN7irCeD41KtVpYXWd)s3pAcR3c6RGCfjhpL3ZlRxl26nY78(MLkCeTTSgWRPqxT2oHyXq59BJFeJplOJxtmkst(QGG67Au9(Mia8O(edziSBdXKmWSBEXl2sLrbzXV5iVTU34zM4TP(2cbBkAa8qr(S4sRKlhin4uO1wg6v2K1fryFaBE4GHcqIuJhn6vUXOquK575Yg5kYffmjg((Rp(OHOW4WV)1Xem)q)RuzZXtkqUxnEJA0dpTwkFIaWux4YENRIkiCIJQEBD9ZSR8SGdC21hHjMcEOZ8EwvYslJBPNU(OfZMxjd7rQwSt91DUMBUxN4h5OaJalHjvPI6Dgq2NF2vCR)wjtTMDL3FeAWjA8lzv9NLZLf5BQlitJJokb8x3MEn91xsFLoXrFe(ATw833HEVioTU(M5PhY7VbNHPLCiYDkqDoShelllwsf8r6tMND4(1Mar86sLDmOBxqYVZ8th7SmHvxSSCiIpnh2mT17hDDtpu3SEAuODASBdPMrU3GoyhgEeYxOhrWXgp2tFdiFdDaZQx3MC2f887mmn))j1QjyGAOVJeDIZt(bD5j6azgpdBkmmQchCMmtGi4nOUQ84qg7K1iCsyIh8ZSpTNjIRkpGS4g3W0fUp9C8qYeMJsKFWuc77(s7EqdjPSJUBzYKP40KeDdoSMyydkAOXdvDgJ98cqCSXECOWd49fgQ6lJt8cp(pmgi0P2SGBc68G0KQBY8wAQuQzYeKOIWZmZzGyXQh)hAs2miq4knQ3O1U7JDdmAoZ2APWGm(j1L5554ZJTV4ukk9jseH8j6teQLs91qRaO29JWQlLOW4l0(ZcZcGV7fzYj1mbPidQDAUP4lJYKAhm3xyQhSb8ljs2VC4Exp3uIvkzS0ilQuRIyyYygiZmm5utpmmgfFgVZJ9BV633dxEjLPS5PCbm5WbrgPErqrugyq2He8IgFiWYKW6iZZsJWx0mzJA9HQOFYOufkJeYO5JE6yiESWVF5DH3FFgMAPcIJ5XUMFA1W)0cvDmwmRgoyWtrswig7s(fvrYqul)3Z5BrNu5BH0gLVfcXTT5(rXTn773GebynRlgHCI6A6qSgfqWp7xhfjnqP0GFoSnKbD5bas9EQUEkeIh8OOJeEgHTwmWR5fnuAB6o5gJ12EtVv12aVNfUq9tOu3dDCrhvUdSEfVrx4rWfznkDGvtgSyYMxNww3o(NoQR2YMXS5nmAWjUwikvm3BkgRBhb1mkxK3Jft9CJ8ES4ZNiE4SSdyDRCduY8PYGRKe8mmVc9ebp)SUNwAbR9PK5msH10QWkOqSGjtZeAgQbBM6dVoLOsOMusTIJPWDPSkRqn7lu1kHXtKg1FFoAA)9pCt2sz4AmZutPkCGE0fzFwXwQEVnvltJlF8nz7(KqWX0nn6W7gOwAbQt(uKFVFrHY)WRq5MOpj)wD6ZBE7sEOTKAs2jFnjJL0Fp9QvM91Nvrkx(iPn4rgAQlxRJ2LqDBivFhWf57QgCGjyExVET6TAkK7VnJCipu0E1BT)(YnAjFqtjP95GI7C7GixgiZl4qeSnDJl5Ix9l6gw(QjIKC49Lewhk4yHTKoBsjBtyqB4cYj94hL8mskepHKeybF9pFHLNbJLnkYaIXfifrxmQaHrNqmabDIhnGcikUsWhtWpgHFOIF4kJVd)ye(HQCcIvW1uaMaQSdvBmALm9jD0ZNBwLDqV4prh12CaNTXDr70IaL1ipt7kupXoA3svFUC1Vn7jpxj2mxmibOsfxzclCLWR5QCIo3K8A8ewJNuuJhLxJhXA8OIASAEnwL1yvPQzZ5jeFN8wVgS8K4XEISvlW6S1hYJ4w1DFSMRZdZAo(fddt57Isl2q7YfS9ReButw59UrV8NEtLtBw9yJhpR8KxJvEzMxZLjfQJBixG)VJLBZzCf9bCNPtESBJoMRHx7CzDdVG3Ywx6BAv7ytBD8Hk5hY8qzwI3EvXV8Uw31ifZqOy4v9sdKWngcHcagM7Nlmo)BNFPpJzOYZkQkStv)QHC3t)S0vva3ZWzZ0uX3(eCb(vcZSFqrXm4gKTilSY4WDYLFuCOckJfXbhzt0QNyNxZmmhpUzN)EmsLrIr(BOGv(7rRBJVZ2Ao5Qn(YbxnaTRjQk)E72syB1SpOF)V)h)l)p)L)8Vr28kDpS5TetRZ)JNBm(V(p)Z)1)Y)()9)0)2F9F9F(Z4XogV6L3KzWgp8WG1(qLPMOo2)omay)g72IYOW9AT5i6K295lcBbwDL0O)G3tMxCozswAG)zoZ8Esn4sbKrXUTQJzjLyypZXcS0UlMSTQr9J1(b5gVsIYwSiBRsysv2Mwg72jxm6vClydsRLsW3jAtelHkFYtEWLTVE0hDAp)NFU0tMA3zOK7ydFTPb)4w0PiLVlovCkJY3mnwzxFRXcKwM61l3A3wPUay4HsBZv8UqShmEwdkBkFq3qQKxppXuRETPnwQyCfVBhLapsCH2eJ4Lmysjk0)ZIAWHslK(saEkTMrTDnrHN9vDPF3z(Tqpx5Mpn65YplcYFzs9rFsfMnY7Yj5PsL3Sue9bfPU6zB8xmvR7epsDXvVnT6VBQcVwGeAlb5QF)leUrWUXgvxGFRDq1mMAX5ETDe(MofPYCW95HCudHVZBcR3uKW7HK7T60PvPkPZ8c8KFOoZLj7Qev9QEUAjEKR5Orw0bIVN9nZD0PZ6S2nuc(UrUHpB94fc0OqeXkmiJXpLp)Tsqe7uwJigRIUOmkkHIbfuiWJIHDTvgwfK8qcTmwRsBeYt2pTvFzg8ZZmyqyQwo(af1L3nD2EYXmLsm0pbM(vQqubDk(tIhkZKf4j6hNhH7Hh6x8OniovnYkTelCVI4TxQMWw1bBTTXqroouk6AhsMrb(UxioSty7aN00Ov(eIKPoDwufIFAnxvsr0Vch5s5nYzT)WaYxZ7fqsoj06EbLmMey7KkyzAAJ(HUBxS72qBZh9BtaWCnqzRWwKckG8DfNcJcwFh2AY6XmVuulWg1OwpkVGof361WlkEy27G9B0AVYhcjY5cvZYayxynIsiGvo1ygEX5qh2ZtwOzc6mCeuLzEckLfgdSy(xOGd40AZDt6Eykv4920SZ8ZDouEVnXE(xYHYNMCOSatY2BJmj73UtNcF3)m8M6FW)1tnth0UJXRAD0rotvqh0gykvztAO78V4Gz9B6V)G8YuO8TTkx5KRgQERQPftpKCTYkPDIz6OErlbZrzX8mqmxpVpkHddbWS3vdk8xq5NmVRhBY7r83kzXnIUFhl4m2PDTPmsKlNftbKYSPCHBp(OT)C6T(fVDbYlNxc3zUShG3NAFC35lBa)LnGFc3aoH4)sqKtRHy4Ejr7if3aMhHtqzbBaD4rp0DEOThLp9NV7t3PE5p28Gc2)j)i8(hB9vi2RBXlQCzKl5Q)Vp`

**Comment by Scabbrox** [comment]
or this: https://wago.io/pZfxzrFaK

**Comment by Scabbrox** [comment]
All the parts other than Lucid Dream behave as expected, I'm at a loss as to why just the essence doesn't, but then again in the pack I linked first in error they don't go on CD for me either, just countdown and stick to screen even when on CD.

I have tried deleting my settings and all of WA and reinstalling fresh but this aura won't work like it did yesterday.

**Comment by mrbuds** [comment]
You must check "Exact Spell Match"

**Comment by Scabbrox** [comment]
In 'Trigger' for both auras?  I have done that but it makes no difference.

**Comment by mrbuds** [comment]
I can't log into retail without getting an hour long queue when going back to classic right now, but the first aura you linked for lucid dream was working well, so there is no bug in weakauras for me.

**Comment by mrbuds** [comment]
Btw it should not be done with 2 auras, but only 1 with show = always and conditions to changed the look of it.

**Comment by Scabbrox** [comment]
I'll see if that makes any difference but I've been using this WA without issue since more or less 8.0 until today.  But then again the aura above didn't work as displayed for me either, I get no cooldown swipe on any essence based auras, only on the trinkets and azeroth traits.

If it's really only me then how can I reset my WA installation to remove all possibilities of some sort of corruption and reinstall afresh as I tried deleting the addon, removed all saved variable files with  weakauras in the folder name and still get the same issue.

**Comment by Scabbrox** [comment]
OK, did some troubleshooting and it's definitely the latest update of Weak Auras that has created the issue.

Firstly I tried reverting my retail installation to a back-up from this morning from before I ran addon updates from my Curse client.

Then I ran the game **without** updating the 4 addons listed (one of which was Weak Auras 2.17.0zip (from 2.16.6)

I also added the https://wago.io/KKcfXqr0U weak aura into WA 2.16.6 as a test against my own aura. 

The weak auras all worked normally as they did yesterday.  Please see the first half of the video I have linked.

Then I exited the game and updated only the weak auras addon to 2.17 and re-ran the test.

See second half of video.

As you will notice in the first part both weak auras worked as they should with icons updating normally and on cool down durations shown as well.

In the second test neither addon worked correctly anymore.

I hope this enables you to track down what functionality has been broken between the two updates.  For now I will revert to 2.16.6 until I see a fix but I'd remove the closed flag on this as it isn't.

Please see video below.

[https://youtu.be/8_6NRKU3h2o](url)

**Comment by Scabbrox** [comment]
https://youtu.be/8_6NRKU3h2o 

as link above doesn't work.

**Comment by mrbuds** [comment]
This look like an issue with your skinning addon.

Can you try with only WeakAuras enable as it was asked in the template ?
What are you using for skinning weakauras, and which version of the addon is it ?

**Comment by Scabbrox** [comment]
Well, I'm using Masque but that wasn't updated today.  I use Elvui with Shadow&Light and S&L did have an update today.  Will revert again and disable S&L and then masque and see if either fix the issue.  It's not just the skinning that's broken though, note the is no timer on the icons in the second part of the video as to when it comes off of CD.

**Comment by Scabbrox** [comment]
Tried everything suggested, including turning off **every** addon except Weak Auras 2 and it's still not working.

I've spent the day on this now so 2.16.6 it is for me until I hear the issue has been located.  If it's a conflict I'm not sure what it can be conflicting with with everything else off.

With all updated and just WA2 at 2.16.6 it works, can't get 2.17.0 to work with these auras no matter what I try.
